cmd/server/handler_oauth2_factory.go

```diff
@@ -56,7 +56,11 @@ func injectFositeStore(c *config.Config, clients client.Manager) {
 
 func newOAuth2Provider(c *config.Config, km jwk.Manager) fosite.OAuth2Provider {
 	var ctx = c.Context()
-	var store = ctx.FositeStore
+	var store = oauth2.CommonStore{
```

Comment:
Can we try to use plugin system here?

Reply:
Unfortunately no, because Hydra's plugin system is only for storages, but not all the code (e.g. we can't add custom grant-type handler via a plugin).